api-flask/app/database/user_info_model.py

```diff
@@ -0,0 +1,26 @@
+"""Alert database model."""
+import datetime
+import json
+import logging
+
+from sqlalchemy import JSON, TIMESTAMP, Column, DateTime, Identity, Integer, String
+from sqlalchemy.ext.declarative import DeclarativeMeta, declarative_base
+from sqlalchemy.sql.sqltypes import Boolean
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+Base = declarative_base()
+
+
+class UserInfoModel(Base):  # type: ignore
```

Comment:
@wadhwamatic can you think of other useful info/columns we might want to keep in here?
`country` maybe? noting that we want to limit usage of this functionality, but it might come in handy still.

I made "access" a json so that we can use this table for other config/limitations in the future